File: prototype/src/data/accountIcons.ts
/** Иконки счетов — emoji, единый набор для циклического выбора */
export const ACCOUNT_ICONS = ['💵', '💳', '🏦', '💰', '🧾', '🏠'] as const

export const DEFAULT_ACCOUNT_ICON = '💳'

export function nextAccountIcon(current: string): string {
  const idx = ACCOUNT_ICONS.indexOf(current as (typeof ACCOUNT_ICONS)[number])
  if (idx === -1) return DEFAULT_ACCOUNT_ICON
  return ACCOUNT_ICONS[(idx + 1) % ACCOUNT_ICONS.length]
}

File: android/app/src/main/java/com/famly/app/domain/AccountIcons.kt
package com.famly.app.domain

val ACCOUNT_ICONS = listOf("💵", "💳", "🏦", "💰", "🧾", "🏠")

const val DEFAULT_ACCOUNT_ICON = "💳"

fun nextAccountIcon(current: String): String {
    val idx = ACCOUNT_ICONS.indexOf(current)
    if (idx == -1) return DEFAULT_ACCOUNT_ICON
    return ACCOUNT_ICONS[(idx + 1) % ACCOUNT_ICONS.size]
}
